packages/near-membrane-base/src/__tests__/environment.spec.ts

```diff
@@ -20,6 +17,36 @@ describe('VirtualEnvironment', () => {
             new VirtualEnvironment();
         }).toThrow(/Missing VirtualEnvironmentOptions options bag/);
     });
+    it('forwards support { ... } as bit fields to init functions', () => {
+        // Ignoring "Property 'assertions' does not exist on type '{...}'."
+        // @ts-ignore
+        expect.assertions(2);
+
+        // @ts-ignore
+        const interceptor: typeof init = (
+            _name: string,
+            _shouldOrNotTrapMutation: boolean,
+            supportFlags: SupportFlagsField,
+            exportsCallback: HooksCallback
+        ) => {
+            expect(supportFlags).toBe(0b1);
+            // @ts-ignore
+            exportsCallback();
+            return () => {};
+        };
+
+        const distortionCallback: DistortionCallback = (v: any) => v;
```

Comment:
`distortionCallback` is optional and we can just let the default one fly (which is essentially what you're passing)